Core/MIPS/JitCommon/JitCommon.cpp

```diff
@@ -157,10 +159,12 @@ const char *ppsspp_resolver(struct ud*,
 		*offset = addr - (uint64_t)MIPSComp::jit->GetBasePtr();
 		return "jitcode";
 	}
+#if defined(_M_IX86) || defined(_M_X64) || defined(MIPS)
```

Comment:
IBM is in the process of switching POWER over to little endian, so maybe if new PowerPC consumer hardware pops up in the future, that won't be an issue anymore :)

SPARC is probably going to be the last arch stuck in big-endian land as all the others have thus soon switched, and I don't see much interest in running PPSSPP on SPARC...